tests/setup.py

```diff
@@ -57,6 +57,7 @@
         "lz4==4.4.3",
         "databricks-sdk==0.49.0",
         "databricks-sql-connector==4.0.2",
+        "duckdb==1.3.1",
```

Comment:
curious if this is sufficient for CDT? I was thinking those dependencies were specified elsewhere, but i realize lots of work has been going on w.r.t. how that works recently.

Reply:
Yes, this is sufficient.